storage/tests/tests/test_set_get.py

```diff
@@ -13,8 +14,57 @@
 ]
 
 
-def test_set_get():
-    sc, sp = common.init(unlock=True)
+@pytest.mark.parametrize("nc_class", NC_CLASSES)
+def test_set_delete(nc_class):
```

Comment:
Just for information: You can create a fixture and parametrize the fixture instead of the test:
```python
@pytest.fixture(params=NC_CLASSES)
def nc_class(request):
    return request.param

def test_set_delete(nc_class):
```

Feel free to resolve this conversation yourself.